tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/CouchbaseCommon.cs

```diff
@@ -0,0 +1,128 @@
+// <copyright file="CouchbaseCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase
+{
+    internal static class CouchbaseCommon
+    {
+        internal const string CouchbaseClientAssemblyName = "Couchbase.NetClient";
+        internal const string CouchbaseOperationTypeName = "Couchbase.IO.Operations.IOperation";
+        internal const string CouchbaseConnectionTypeName = "Couchbase.IO.IConnection";
+        internal const string CouchbaseOperationV3TypeName = "Couchbase.Core.IO.Operations.IOperation";
+        internal const string CouchbaseConnectionV3TypeName = "Couchbase.Core.IO.Connections.IConnection";
+        internal const string CouchbaseGenericOperationTypeName = "Couchbase.IO.Operations.IOperation`1[!!0]";
+        internal const string CouchbaseOperationResultTypeName = "Couchbase.IOperationResult<T>";
+        internal const string MinVersion2 = "2.2.8";
+        internal const string MaxVersion2 = "2.7.25";
+        internal const string MinVersion3 = "3";
+        internal const string MaxVersion3 = "3";
```

Comment:
Does this need to be `3.*.*`? (I'm not sure, this changed with call target instrumentation.)

Reply:
It works locally and it is what has been done for ES7. So I assumed it was ok to do that (I initially went for 3.x.x and changed)